### PR TITLE
[trvl-px.com] New ruleset

### PR DESCRIPTION
**Pull request by youdly**

### CODE REVIEW

**Comment by J0WI** [comment]
This seems related too `Expedia.xml`. Could you please add the references?

**Comment by youdly** [comment]
Added references.

File: src/chrome/content/rules/trvl-px.com.xml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+<!--
```
Comment (J0WI) [inline src/chrome/content/rules/trvl-px.com.xml]: We use to add a reference back to the 'parent' ruleset too. (Same for pending #13976 and #13977.)

Comment (youdly) [reply src/chrome/content/rules/trvl-px.com.xml]: Added reference to `Expedia.com.xml` which I'll rename from `Expedia.xml` in separate PR. Current working on its ccTLD.

**Pull request merged**